bodhi/tests/server/test_push.py

```diff
@@ -305,7 +305,13 @@ def test_abort_push(self, publish):
 
         # The exit code is 1 when the push is aborted.
         self.assertEqual(result.exit_code, 1)
-        self.assertEqual(result.output, TEST_ABORT_PUSH_EXPECTED_OUTPUT)
+
+        # This is a terribly dirty hack that strips an SQLAlchemy warning about calling configure
+        # on a scoped session with existing sessions. This should ultimately be fixed by making
+        # sure there are no sessions when the CLI is invoked (since it calls configure)
+        doctored_output = result.output.split('\n', 2)[2]
+        doctored_output = result.output
```

Comment:
@pypingou Good point, I'll make a change to search for a portion of this message and only filter if it's present. Also, I'm not sure how the tests could possibly pass with the above line and I'm not sure how I didn't notice that last week so I'll fix that too.